docs/sql/got-tutorial/sql-crud.md

```diff
@@ -0,0 +1,154 @@
+---
+sidebar_position: 10
+title: Part 1 - Basic Document Tasks
+---
+
+You can perform basic document tasks such as create, read, update, and delete (CRUD) on documents in a collection. This portion of the tutorial guides you through those tasks.
+We will firs review the syntax of CRUD commands and then use SDKs to send queries to Macrometa platform.
+
+## Create the Characters Collection
+
+Before we can insert documents with SQL, we need a place to put them in: a collection.
+
+For this tutorial, [Create a Document Store collection](../../collections/documents/create-document-store.md) in the console. For more information about collections, refer to [Collections](../../collections/index.md).
+
+1. [Log in to your Macrometa account](https://auth.paas.macrometa.io/).
+2. Click **Collections**.
+3. Click **New Collection**.
+4. Click **Document Store**.
+5. Name the collection **categories** and then click **Create**.
+
+### Add One Document to the Collection
```

Comment:
I suggest adding a section starting with an h2 above this. The transition from creating the collection to doing things to documents in it is very abrupt, I think the reader would benefit from some context so that they know what they'll be doing and why. 

Since this is a tutorial, maybe lean on the dataset and talk about how we're managing inventory for a small online store or something. Give me some context to work on.

With the GoT tutorial, we were exploring the complex family relationships in Game of Thrones. That was interesting, because you really do need a diagram for some of those. This is a much simpler example, which is good, but it means that you'll have a harder time keeping the student's interest.

Tell me WHY I am doing these things. Show me how this can be useful on a small scale so that I can imagine how I could apply it to my real-world use case.